Form/Extension/ParsleyTypeExtension.php

```diff
@@ -141,6 +142,10 @@ public function finishView(FormView $view, FormInterface $form, array $options)
         $constraints = $this->constraintBuilder->build();
 
         foreach ($constraints as $constraint) {
+            if (!$constraint instanceof NormalizableInterface) {
```

Comment:
I check for the NormalizableInterface here because I don't want my constraints to extend the constraint of this bundle. I simply want to check if the constraint can be normalized.

If a user of the bundle wants to remove it from their project they would have to change all the extends of their constraints? This would be a type of vendor locking.